modules/ROOT/pages/getting-started.adoc

```diff
@@ -10,21 +10,8 @@ include::{partialsdir}/registering-a-mobile-app.adoc[leveloffset=1]
 
 Mobile Services provide commonly required features for mobile app development.
 
-The following services are available with this release:
+This guide introduces the procedures for using mobile services by guiding you through the process using the Identity Management service. For a full list of available services see xref:con_services.adoc[Mobile Services]
 
-
-* xref:identity-management.adoc[{keycloak-service}]
-* xref:push-notifications.adoc[{unifiedpush-service}]
-* xref:runtime-connector.adoc[{custom-service}]
-* xref:mobile-metrics.adoc[{metrics-service}]
-* xref:mobile-cicd.adoc[{digger-service}]
-* xref:device-security.adoc[{device-security-service}]
-
-
-This guide introduces the procedures for using mobile services by guiding you through the process for running an Android app using the Identity Management service. 
-
-include::keycloak/provisioning.adoc[leveloffset=1]
```

Comment:
Removing this include removes the section on how to do the provisioning so we still need this here.